### PR TITLE
Fixes cult shifter not spawning from archives

### PR DESCRIPTION
**Pull request by scoopscoop**
:cl:
fix: The cult shifter will now properly spawn as part of the veil walker set.
/:cl:

Hope I didn't fuck anything up, I haven't touched github in ages.

### CODE REVIEW

File: code/game/gamemodes/cult/cult_structures.dm [hunk 1]
```diff
@@ -207,19 +207,20 @@ var/list/blacklisted_pylon_turfs = typecacheof(list(
 		user << "<span class='cultitalic'>The magic in [src] is weak, it will be ready to use again in [getETA()].</span>"
 		return
 	var/choice = alert(user,"You flip through the black pages of the archives...",,"Supply Talisman","Shuttle Curse","Veil Walker Set")
-	var/pickedtype
+	var/pickedtype[0]
```
Comment (ChangelingRain) [inline code/game/gamemodes/cult/cult_structures.dm]: This should probably be `var/pickedtype = list()`; I'm not sure there's a difference, but this is a rather rare way to do it.

File: code/game/gamemodes/cult/cult_structures.dm [hunk 2]
```diff
@@ -207,19 +207,20 @@ var/list/blacklisted_pylon_turfs = typecacheof(list(
...
+			pickedtype += /obj/item/weapon/paper/talisman/supply/weak
 		if("Shuttle Curse")
-			pickedtype = /obj/item/device/shuttle_curse
+			pickedtype += /obj/item/device/shuttle_curse
 		if("Veil Walker Set")
-			pickedtype = /obj/item/device/cult_shift
-			pickedtype = /obj/item/device/flashlight/flare/culttorch
+			pickedtype += /obj/item/device/cult_shift
+			pickedtype += /obj/item/device/flashlight/flare/culttorch
 	if(src && !qdeleted(src) && anchored && pickedtype && Adjacent(user) && !user.incapacitated() && iscultist(user) && cooldowntime <= world.time)
```
Comment (phil235) [inline code/game/gamemodes/cult/cult_structures.dm]: pickedtype.len

**Review by phil235 — APPROVED** [review]
:+1:

File: code/game/gamemodes/cult/cult_structures.dm [hunk 3]
```diff
@@ -207,19 +207,20 @@ var/list/blacklisted_pylon_turfs = typecacheof(list(
 		user << "<span class='cultitalic'>The magic in [src] is weak, it will be ready to use again in [getETA()].</span>"
 		return
 	var/choice = alert(user,"You flip through the black pages of the archives...",,"Supply Talisman","Shuttle Curse","Veil Walker Set")
-	var/pickedtype
+	var/pickedtype = list()
```
Comment (phil235) [inline code/game/gamemodes/cult/cult_structures.dm]: var/list/pickedtype = list()
Otherwise the code doesn't know it's a list

**Pull request merged**